docs/topics/appendix-glossary.adoc

```diff
@@ -0,0 +1,29 @@
+
+[[glossary]]
+= Glossary
+
+== Product and Project Names
+
+{launcher}:: The web application you launch boosters with.
+
+{OpenShiftLocal}:: An OpenShift cluster running on your machine using Minishift.
+
+== Terms Specific to {launcher}
+
+[glossary_booster]
+Booster:: An language-specific implementation of a particular xref:glossary_mission[mission] on a particular xref:glossary_runtime[runtime]. Boosters are listed in a xref:glossary_booster_catalog[booster catalog].
++
+For example, a booster is a web service with a REST API implemented using the {WildFlySwarm} runtime.
+
+[[glossary_booster_catalog]]
+Booster Catalog:: An Git repository that contains information about boosters.
```

Comment:
Is that "a Git repository" instead of "an"?

Reply:
Ha, good catch, thanks.